### PR TITLE
[MM-21277] Update Google Apps Oauth 2 Steps

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary

Updated description to include enabling peoples API

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-21277

#### Screenshots

image

### CODE REVIEW

**Comment by hahmadia** [comment]
> The formatting for the word "Enable" is different in steps 7 and 8 (Google People API and Google+ API), but they should match. One is bold and one is italic, but they should both be italic (what it was before).

Thanks for the catch. Also noticed I hadn't added a period at the end of the sentence. I have updated the main post with updated photo.

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -3379,7 +3379,7 @@ const AdminDefinition = {
...
...                              help_text_default: '1. [Log in](!https://accounts.google.com/login) to your Google account.\n2. Go to [https://console.developers.google.com](!https://console.developers.google.com), click **Credentials** in the left hand sidebar and enter "Mattermost - your-company-name" as the **Project Name**, then click **Create**.\n3. Click the **OAuth consent screen** header and enter "Mattermost" as the **Product name shown to users**, then click **Save**.\n4. Under the **Credentials** header, click **Create credentials**, choose **OAuth client ID** and select **Web Application**.\n5. Under **Restrictions** and **Authorized redirect URIs** enter **your-mattermost-url/signup/google/complete** (example: http://localhost:8065/signup/google/complete). Click **Create**.\n6. Paste the **Client ID** and **Client Secret** to the fields below, then click **Save**.\n7. Go to [Googles People API](!https://console.developers.google.com/apis/library/people.googleapis.com) and click *Enable*.\n8. Finally, go to [Google+ API](!https://console.developers.google.com/apis/api/plus/overview) and click *Enable*. This might take a few minutes to propagate through Google`s systems.',
```
Comment (etoaster) [inline components/admin_console/admin_definition.jsx]: Change the ` in "Google`s" (almost at the end of the line) to a '

File: components/admin_console/admin_definition.jsx [hunk 2]
```diff
@@ -3379,7 +3379,7 @@ const AdminDefinition = {
...
...                             help_text_default: '1. [Log in](!https://accounts.google.com/login) to your Google account.\n2. Go to [https://console.developers.google.com](!https://console.developers.google.com), click **Credentials** in the left hand sidebar and enter "Mattermost - your-company-name" as the **Project Name**, then click **Create**.\n3. Click the **OAuth consent screen** header and enter "Mattermost" as the **Product name shown to users**, then click **Save**.\n4. Under the **Credentials** header, click **Create credentials**, choose **OAuth client ID** and select **Web Application**.\n5. Under **Restrictions** and **Authorized redirect URIs** enter **your-mattermost-url/signup/google/complete** (example: http://localhost:8065/signup/google/complete). Click **Create**.\n6. Paste the **Client ID** and **Client Secret** to the fields below, then click **Save**.\n7. Go to [Googles People API](!https://console.developers.google.com/apis/library/people.googleapis.com) and click *Enable*.\n8. Finally, go to [Google+ API](!https://console.developers.google.com/apis/api/plus/overview) and click *Enable*. This might take a few minutes to propagate through Google\'s systems.',
```
Comment (bradjcoughlin) [inline components/admin_console/admin_definition.jsx]:
```suggestion
                                help_text_default: '1. [Log in](!https://accounts.google.com/login) to your Google account.\n2. Go to [https://console.developers.google.com](!https://console.developers.google.com), click **Credentials** in the left hand sidebar and enter "Mattermost - your-company-name" as the **Project Name**, then click **Create**.\n3. Click the **OAuth consent screen** header and enter "Mattermost" as the **Product name shown to users**, then click **Save**.\n4. Under the **Credentials** header, click **Create credentials**, choose **OAuth client ID** and select **Web Application**.\n5. Under **Restrictions** and **Authorized redirect URIs** enter **your-mattermost-url/signup/google/complete** (example: http://localhost:8065/signup/google/complete). Click **Create**.\n6. Paste the **Client ID** and **Client Secret** to the fields below, then click **Save**.\n7. Go to the [Google People API](!https://console.developers.google.com/apis/library/people.googleapis.com) and click *Enable*.\n8. Finally, go to [Google+ API](!https://console.developers.google.com/apis/api/plus/overview) and click *Enable*. This setting might take a few minutes to propagate.',
```
Just a suggestion to respect the Google trademark [rules](https://www.google.com/permissions/trademark/rules/) a bit better.

File: i18n/en.json [hunk 3]
```diff
@@ -642,7 +642,7 @@
...
+  "admin.google.EnableMarkdownDesc": "1. [Log in](!https://accounts.google.com/login) to your Google account.\n2. Go to [https://console.developers.google.com](!https://console.developers.google.com), click **Credentials** in the left hand sidebar and enter \"Mattermost - your-company-name\" as the **Project Name**, then click **Create**.\n3. Click the **OAuth consent screen** header and enter \"Mattermost\" as the **Product name shown to users**, then click **Save**.\n4. Under the **Credentials** header, click **Create credentials**, choose **OAuth client ID** and select **Web Application**.\n5. Under **Restrictions** and **Authorized redirect URIs** enter **your-mattermost-url/signup/google/complete** (example: http://localhost:8065/signup/google/complete). Click **Create**.\n6. Paste the **Client ID** and **Client Secret** to the fields below, then click **Save**.\n7. Go to [Googles People API](!https://console.developers.google.com/apis/library/people.googleapis.com) and click *Enable*.\n8. Finally, go to [Google+ API](!https://console.developers.google.com/apis/api/plus/overview) and click *Enable*. This might take a few minutes to propagate through Google's systems.",
```
Comment (bradjcoughlin) [inline i18n/en.json]: If the previous suggestion looks okay, apply here of course : )

**Comment by hahmadia** [comment]
@etoaster 
I've updated the text as per Brad's suggestion for respecting Google's trademark policy. Could you re-review and make sure all is good? Thanks

**Pull request merged**